vector/src/main/res/drawable/sticker_description_background.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle">
+    <corners android:radius="10dp" />
+    <solid android:color="@color/list_divider_color_light" />
```

Comment:
did you check with theme update ? ie black / dark theme ?

Reply:
Thanks to point that. I will check.